google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/CookiesHolderTest.java

```diff
@@ -185,7 +188,8 @@ public void tearDown() throws Exception {
 
   @Test
   public void testReadRows() {
-    client.readRows(Query.create("fake-table")).iterator().hasNext();
+    @SuppressWarnings("MismatchedQueryAndUpdateOfCollection")
```

Comment:
Maybe easier to just put a class level @SuppressWarnings("ResultOfMethodCallIgnored") ?

I can see wanting more granular suppression, so not blocking, but since it's just a test file I'd personally lean towards the class suppression for conciseness

This applies to a couple of the other classes below as well